forms/utils.py

```diff
@@ -0,0 +1,14 @@
+from django.utils.text import slugify
+
+
+def generate_unique_identifier(klass, field, max_length):
+    origin_identifier = slugify(field)[:max_length]
+    unique_identifier = origin_identifier
+    index = 1
+    while klass.objects.filter(identifier=unique_identifier).exists():
+        unique_identifier = "%s-%d" % (
```

Comment:
[Docs](https://docs.python.org/2/library/stdtypes.html#str.format) suggest that the you should use the `str.format()` style instead of the modulo style when writing new code in Python3.x.

Reply:
Changed